eng/scripts/validate_go_mod.ps1

```diff
@@ -0,0 +1,23 @@
+Param(
+    [string] $serviceDir
+)
+
+Push-Location sdk/$serviceDir
+
+$goModFiles = Get-ChildItem -Path . -Filter go.mod -Recurse
+
+if ($goModFiles.Length -eq 0) {
+    Write-Host "Could not find a go.mod file in the directory"
+    exit 1
+}
+
+foreach ($goMod in $goModFiles) {
+    $patternMatches = Get-Content $goMod.FullName | Select-String -Pattern "replace "
```

Comment:
```suggestion
    $patternMatches = Get-Content $goMod.FullName | Select-String -Pattern " replace "
```

Reply:
Why the space beforehand? The replace directive should start at the beginning of a new line